src/Components/FormCreation/FormSection.js

```diff
@@ -160,6 +178,20 @@ function FormSection({
           ) : null}
         </CardWrapper>
       ))}
+      <Menu
+        id="actions-menu"
+        classes={classes}
+        anchorEl={anchorEl}
+        getContentAnchorEl={null}
+        anchorOrigin={{ vertical: "bottom", horizontal: "right" }}
+        transformOrigin={{ vertical: "top", horizontal: "center" }}
+        keepMounted
+        open={Boolean(anchorEl)}
+        onClose={handleAnchorClose}
+      >
+        <MenuItem onClick={console.log("b")}>Move section</MenuItem>
+        <MenuItem onClick={console.log("a")}>Delete section</MenuItem>
```

Comment:
I hope the console log statements do not fail the travis CI checks. If travis CI passes, I do not mind keeping these since they will be overwritten.